composeexample/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for composeexample project.
+
+Generated by 'django-admin startproject' using Django 2.2.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.2/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '9(!tz^8yeix#k64$&r6^_w*#53vjmrsttu!nh%2u#)s@8yn9vp'
```

Comment:
Are we OK checking this in?

Reply:
it will have to bee changed before it goes into production.
but will be fine for development.